internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -294,11 +295,11 @@ func (h *Host) CreateContainer(ctx context.Context, id string, settings *prot.VM
 	// containers can have access to it. The security policy is required by containers which need to extract
 	// init-time claims found in the security policy.
 	//
-	// We append the variable after the security policy enforcing logic completes so as to bypass it; the
+	// We append the variable after the security policy enforcing logic completes to bypass it; the
 	// security policy variable cannot be included in the security policy as its value is not available
 	// security policy construction time.
-	if policyEnforcer, ok := (h.securityPolicyEnforcer).(*securitypolicy.StandardSecurityPolicyEnforcer); ok {
-		secPolicyEnv := fmt.Sprintf("SECURITY_POLICY=%s", policyEnforcer.EncodedSecurityPolicy)
+	if val, ok := settings.OCISpecification.Annotations[annotations.SecurityPolicyEnv]; ok && strings.ToLower(val) == "true" {
```

Comment:
nit: could we use one of the oci annotations functions we have to parse this as a bool?

Reply:
the function that parses the bool isn't exported currently. Do you mind if I work on it in a follow up PR instead, since it'll touch a bunch of unrelated code.

Reply:
turned out to be not that bad